client/blocks/action-card/README.md

```diff
@@ -0,0 +1,105 @@
+Action Card
+=========
+This is a card component that has a call to action button
+
+#### How to use:
+
+```js
+import ActionCard from 'blocks/action-card';
+
+render: function() {
+  return (
+	<ActionCard
+		headerText={ 'Header' }
+		mainText={ 'Some text' }
+		buttonText={ 'Call to action!' }
+		buttonIcon="external"
+		buttonPrimary={ true }
+		buttonHref="https://wordpress.com"
+		buttonTarget="_blank"
+		buttonOnClick={ noop }
+	/>;
+  );
+}
+```
+
+## Props
+
+Below is a list of supported props.
+
+### `headerText`
+
+<table>
+	<tr><td>Type</td><td>String</td></tr>
+	<tr><td>Required</td><td>Yes</td></tr>
+</table>
```

Comment:
We could use the [Markdown syntax for tables](http://www.tablesgenerator.com/markdown_tables) which in addition to be easier to read is also supported by the DevDocs section (otherwise you would just see [HTML tags](http://calypso.localhost:3000/devdocs/client/blocks/calendar-button/README.md) instead of the table).

I think an even better approach would be to get rid of those tables though, since it's just duplicating information from component's prop types (and there is a good chance they will go out of sync at some point in the future). [Some components](http://calypso.localhost:3000/devdocs/client/components/foldable-card/README.md) already follow that pattern.